includes/Modules/Idea_Hub.php

```diff
@@ -181,6 +182,18 @@ public function register() {
 			 * Show admin notices on the posts page if we have saved / new ideas.
 			 */
 			add_filter( 'googlesitekit_admin_notices', $this->get_method_proxy( 'admin_notice_idea_hub_ideas' ) );
+
+			/**
+			 * Adds a special class name to idea posts.
+			 */
+			add_filter( 'post_class', $this->get_method_proxy( 'update_post_classes' ), 10, 3 );
+
+			add_action(
+				'admin_footer',
+				function() {
+					echo '<div id="js-googlesitekit-post-list" class="googlesitekit-plugin"></div>';
```

Comment:
Let's also only output this for posts.
```suggestion
					if ( 'post' !== get_current_screen()->post_type ) {
						return;
					}
					echo '<div id="js-googlesitekit-post-list" class="googlesitekit-plugin"></div>';
```

Reply:
Updated.